azext_iot/digitaltwins/command_map.py

```diff
@@ -45,6 +45,31 @@ def load_digitaltwins_commands(self, _):
         cmd_group.wait_command("wait", "wait_instance")
         cmd_group.command("reset", "reset_instance", confirmation=True, is_preview=True)
 
+    with self.command_group(
+        "dt data-history",
+        command_type=digitaltwins_resource_ops,
+        is_preview=True
+    ) as cmd_group:
+        pass
+
+    with self.command_group(
+        "dt data-history connection",
+        command_type=digitaltwins_resource_ops,
+        is_preview=True
```

Comment:
Since the parent `dt data-history`  has `is_preview=True`, we shouldn't need to re-declare it here. We should test/ensure this statement, then could remove it here.

Reply:
![image](https://user-images.githubusercontent.com/73560279/152604804-2f426a26-b913-466a-9b45-7eefef4a32e3.png)